x-pack/plugins/cloud_security_posture/public/common/api/use_package_policy_list.ts

```diff
@@ -21,24 +21,32 @@ interface PackagePolicyListData {
 
 const PACKAGE_POLICY_LIST_QUERY_KEY = ['packagePolicyList'];
 
-export const usePackagePolicyList = (packageInfoName: string) => {
+export const usePackagePolicyList = (packageInfoName: string, enabled = true) => {
```

Comment:
nit
```suggestion
export const usePackagePolicyList = (packageInfoName: string, {enabled }) => {
```
a bit better so later the usage is a bit clearer to read like:
```typescript
const { data: packagePolicyList } = usePackagePolicyList(packageInfo.name, 
    { enabled: canFetchIntegration }
);
```